Source/glTFRuntime/Private/glTFRuntimeParserStaticMeshes.cpp

```diff
@@ -598,34 +588,6 @@ UStaticMesh* FglTFRuntimeParser::LoadStaticMesh_Internal(TSharedRef<FglTFRuntime
 
 		}
 #endif
-
-		if (StaticMeshConfig.bBuildLumenCards)
```

Comment:
Should we not keep this code in? Was there a reason it causes an issue for our PDP viewer?

Reply:
Re-added and it's working fine

Reply:
I think we agreed that this was okay

Reply:
Back in and working